README.md

```diff
@@ -1,24 +1,143 @@
+  <h3><b>BUDGET-APP README</b></h3>
+</div>
 
-Things you may want to cover:
+# 📗 Table of Contents
 
-* Ruby version
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+- [Prerequisites](#prerequisites)
+- [Setup](#setup)
+- [Usage](#usage)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* System dependencies
+# 📖 [BUDGET APP] <a name="about-project"></a>
 
-* Configuration
+**BYDGET APP** is a a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-* Database creation
+## 🛠 Built With <a name="built-with"></a>
 
-* Database initialization
+### Tech Stack <a name="tech-stack"></a>
 
-* How to run the test suite
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+### Key Features <a name="key-features"></a>
 
-* Deployment instructions
+- **Authenticate and Authorize user**
+- **Add a new category**
+- **Present a list of transactions for a selected category**
 
-* ...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+to install git on your computer. Follow the instruction on the page to do so
+[Click here ](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and follow the instruction on the page to install git.
+[Click here ](https://www.ruby-lang.org/en/documentation/installation/) and follow the instruction on the page to install ruby.
+[Click here ](https://guides.rubyonrails.org/v5.1/getting_started.html) and follow the instruction on the page to install rails.
+[Click here ](https://www.postgresql.org/download/) and follow the instruction on the page to install postgres.
+
+### Setup
+
+Open your preferred folder in the terminal
+
+Clone this repository to your desired folder:
+Using the commands below:
+
+```sh
+  git clone https://github.com/emarkees/New_Budget_App.git
+```
+
+Open the project on your code editor
+
+Change directory to the project folder
+
+```sh
+  cd BUDGET APP
+```
+
+### Install
+Then, run the following command to install all dependecies
+
+```sh
+    bundle install
+```
+
+### Usage
+
+Then, run the following command to start the app
+
+```sh
+    rails server
+```
+
+## 👥 Author <a name="author"></a>
+
+👤 **Adeyemi Olusola**
+
+- GitHub: [@adeyemi](https://github.com/emarkees)
+- Linkedin: [@adeyemi](https://linkedin.com/in/adeyemi-olusola)
+
+https://BYDGET APP-service.onrender.com/
```

Comment:
- [ ] I don't think this link should be here so kindly remove this broken link :pray: